android/app/src/main/java/app/organicmaps/search/SearchFragment.java

```diff
@@ -284,12 +313,26 @@ public void onChanged()
     });
     mShowOnMapFab = root.findViewById(R.id.show_on_map_fab);
     mShowOnMapFab.setOnClickListener(v -> showAllResultsOnMap());
+    mShowOnMapFab.addOnLayoutChangeListener(new View.OnLayoutChangeListener()
+    {
+      @Override
+      public void onLayoutChange(View v, int left, int top, int right, int bottom, int oldLeft, int oldTop, int oldRight, int oldBottom)
+      {
+        int height = v.getMeasuredHeight();
+        if (height > 0)
+        {
+          mShowOnMapFab.removeOnLayoutChangeListener(this);
+          view.requestApplyInsets();
+        }
+      }
+    });
```

Comment:
Can we avoid copy-paste with BookmarksListFragment? Take out into some base fragment class or InsetUtils?

Reply:
done